### PR TITLE
Add support to customize committer info

### LINKED ISSUE #19: Allow committer params be customizable
Should the committer params be customizable? Otherwise maybe they want to link those to a service account

_Originally posted by @droidpl in https://github.com/ActionsDesk/ghec-invitations-report-action/pull/18#discussion_r525263528_

### PR DESCRIPTION
**Pull request by droidpl**
Adds two new parameters:
- committer-name
- committer-email

`npm run test`: passed

Closes #19

### CODE REVIEW

**Comment by droidpl** [comment]
@stoe this seems to be working from the tests we did on Friday. Shall we merge it?

File: action.yml [hunk 1]
```diff
@@ -12,6 +12,14 @@ inputs:
   enterprise:
     description: 'GitHub Enterprise Cloud account, will require the token to have admin:org, read:enterprise, read:user, repo, user:email scopes'
     required: false
+  committer-name:
+    description: 'The name of the committer that will appear in the Git history'
+    default: 'invitation-reporter[bot]'
```
Comment (stoe) [inline action.yml:17]: @droidpl, was wondering if we should just go with

```suggestion
    default: 'github-actions'
```

like in https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

File: action.yml [hunk 2]
```diff
@@ -12,6 +12,14 @@ inputs:
...
+    required: false
+  committer-email:
+    description: 'The committer email that will appear in the Git history'
+    default: 'invitation@reporter'
```
Comment (stoe) [inline action.yml:21]: @droidpl, was wondering if we should just go with

```suggestion
    default: 'github-actions@github.com'
```

like in https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

**Comment by droidpl** [comment]
Added your reviewed elements. I agree with the change

**Review by stoe — APPROVED** [review]
LGTM :shipit:

**Pull request merged**